src/main/java/com/hillel/items_exchange/model/User.java

```diff
@@ -53,4 +53,19 @@
 
     @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
     private Set<Child> children;
+
+    @ManyToMany(cascade = {CascadeType.ALL})
```

Comment:
I am confused about behavior that we can observe deleting a user. CascadeType.ALL propagates deleting child entity this behavior is not every needed us. Maybe should not use this type here?